.github/workflows/npm.yml

```
@@ -24,11 +24,21 @@ jobs:
           registry-url: "https://registry.npmjs.org"
           cache: "yarn"
 
+      # We need this step because the `@keep-network/tbtc` which we update in
+      # next step has a dependency to `@summa-tx/relay-sol@2.0.2` package, which
+      # downloads one of its sub-dependencies via unathenticated `git://`
+      # protocol. That protocol is no longer supported. Thanks to this step
+      # `https://` is used instead of `git://`.
+      - name: Configure git to don't use unauthenticated protocol
+        shell: bash
+        run: git config --global url."https://".insteadOf git://
+
       - name: Resolve latest contracts
         run: |
           yarn upgrade \
```

Let's pin to exact dependencies versions. (see https://github.com/keep-network/keep-core/pull/3083 for explaination)

```suggestion
          yarn upgrade --exact \
```